### PR TITLE
Good-looking edit profile page

### PR DESCRIPTION
**Pull request by feeds**

### CODE REVIEW

File: backend/apps/matching/configuration/participant.py [hunk 1]
```diff
@@ -5,7 +5,22 @@
...
+                p.get_signup_layout(ignore_private=True) for p in self.properties
+            ]
+        return self.signup_layout_all
+
+    def view_layout(self):
+        if self.signup_layout_private is None:
+            self.signup_layout_private = [
+                p.get_signup_layout(ignore_private=False) for p in self.properties
+            ]
+        return self.signup_layout_private
```
Comment (bjrne) [inline backend/apps/matching/configuration/participant.py:23]: Maybe we can choose a better name? It is not clear what is private here, private like internal? Private like user-private? admin-private? authenticated-private?

**Review by bjrne — APPROVED** [review]
Names are refactorable :wink:
But nice overall.

File: backend/apps/matching/configuration/participant.py (hunk 1 above)
Comment (bjrne) [reply backend/apps/matching/configuration/participant.py:23]: Well this was intended for this PR 🙈 or at least an explanation in a comment? I myself don't understand what it means 😅

Comment (Baschdl) [reply backend/apps/matching/configuration/participant.py:23]: @feeds

**Pull request merged**